feat(apps): fill incubator with a bunch of new apps

Crow-Control opened the pull request:
**Description**
This PR adds a lot of Apps. 'Nuf said.

**Type of change**

- [x] Feature/App addition
- [ ] Bugfix
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor of current code

**How Has This Been Tested?**
<!--
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
-->

**Notes:**
<!-- Please enter any other relevant information here -->

**Checklist:**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests to this description that prove my fix is effective or that my feature works
- [x] I increased versions for any altered app according to semantic versioning

Commit:
(feat) fill incubator with a bunch of new apps

Commit:
Ornias' cleanup

stavros-k committed:
stavros' cleanup

stavros-k commented:
__Removed__
* formalms - needs DB
* leantime - needs DB
* xbackbone - needs DB

__Didn't remove__
* blog - builtin sqlite - But can use mariadb
* chyrp-lite - builtin sqlite - But can use mysql

stavros-k commented:
While some s6 apps pass test, running as root but with readonlyrootfs enabled.
There are still errors in their logs complaining about it, Apps seem to start and die after a while.
(That's why I disabled readonlyrootfs on `e241f7b`)

stavros-k commented:
Also removed this because depends on external DB
* bookstack

stavros-k commented:
todo app spits an error, found 'related' [issue](https://github.com/nicholaswilde/docker-todo/issues/1) on the image repo, but it's for `armv7`

stavros-k commented:
All s6 apps should be ready. 
Only valheim an error (while passing test) but not sure if it makes app unsuable or not.

Commit:
Ornias' second cleanup

Ornias1993 commented:
@stavros-k Could you please also indicate WHICH DB is needed?
Because now I cannot document them into the catagories I made.

stavros-k commented:
> @stavros-k Could you please also indicate WHICH DB is needed? Because now I cannot document them into the catagories I made.

* formalms - MySQL / MariaDB 
* leantime - MySQL
* xbackbone - MySQL
* bookstack - MariaDB
* blog (sqlite builtin) - MariaDB 
* chyrp-ltie (sqlite builtin) - MySQL

Commit:
Add basic SCALE Compatibility layer and docs

Commit:
Ornias's cleanup 3

stavros-k committed:
Stavros' cleanup 2

Update service's port on the skipped apps (+32 squashed commit)

Squashed commit:

[361963ced] config -> data on some apps

[545c32b55] wiki

[41fb4f518] twtxt

[02169bad9] static

[a9042669c] sqlitebrowser

[f09ce3d76] smokeping

[01d2f9af6] sickgear

[680fd358f] sickchill

[71856480f] shorturl

[ce130ac8a] Remmina

[6410ec2ae] pixapop

[a1ea2a33f] Piwigo

[0b4c52d43] photoshow

[25e9a44fd] novnc

[84954deae] notes

[b3be0b28d] muximux

[43d2b37a5] mstream

[736f9877f] Medusa

[8ef3434d6] Headphones

[05e52284f] Grav

[7ce6f1723] golinks

[b4eee91e6] fossil

[d19b44434] FIlezilla

[926236447] Doublecommander

[6f68e5253] Digikam

[f14d3af21] Davos

[8108c01ea] Code-server

[496991750] Cloud9

[4ad02a99f] Beets

[f2f0565b9] Apache-musicindex

[9bf8de8c4] aria2

[283edf7b0] Audacity

Commit:
Ornias' cleanup 4

try something for amcrest and zigbee (+8 squashed commit)

Squashed commit:

[a199779a9] First part of  leaf2mqtt

[0eb7fea43] cryptfolio

[81c2b9b8b] amcrest readonly default true

[fc62aebb4] setup amcrest2mqtt

[d97f7d1fe] remove empty secret references

[ec0e6b1b3] remove sogo for now (till we have a decent mail server and such)

[9be04c1e7] Adapt to recent serviceExpert changes (includes)

(cherry picked from commit 98acc5f93f9f1aa5129942af7f196a04ed6a8c1e)

[8362ebc89] Adapt to recent ingress changes in questions.yaml (includes)

(cherry picked from commit d307ba98a74d470c45c8fdff51423930826015a9)

stavros-k committed:
Stavros' cleanup 3

leaf2mqtt actually needs root when the core fuctionality env's are populated (+6 squashed commit)

Squashed commit:

[4ef306f0c] minio-console

[6ad206b48] finish whoogle and move fixed envs on top

[aaf48575c] finish 4 apps

[97ad3fd4d] don't leave sercrets empty

[e27b98841] sync more env's and sercrets

[398929040] sync secrets

Ornias1993 commented:
@stavros-k Rebased and fixed.

Commit:
Initial work on valheim (WIP)

Ornias1993 commented:
TODO:

- valheim: Second ingress reference for SUPERVISOR webservice
- zigbee2mqtt
- move stateless app to rolling.
- final checkup

Commit:
fix some linting issues

Commit:
Use the configuration only as a default and let users expand it using env vars

Commit:
indent issue on valheim

Commit:
add second ingress to valheim

Ornias1993 commented:
TODO:

- move stateless app to rolling.
- final checkup

Commit:
remove some old valheim comments

Commit:
valheim fix port ints

Ornias1993 commented:
Merging, as I expect the VAST majority of apps to either be working or require just minor touchups. Which are easier to do in a seperate PR.

truecharts-admin commented:
This PR is locked to prevent necro-posting on closed PRs. Please create a issue or contact staff on discord if you want to further discuss this
